reports/New York.md

```diff
@@ -401,6 +401,104 @@ id: ny-newyorkcity-33
 * https://twitter.com/greg_doucette/status/1268189319002537989
 
 
+### Police officer shoves protestor | May 30th
+
+Footage from a Union Square protest shows an officer using his baton to shove a protestor.
+
+tags: shove, baton, protestor
+
+id: ny-newyorkcity-87
+
+**Links**
+
+* https://twitter.com/annikahouse/status/1266931806563176451
+
+
+### Police shove protestors | May 30th
+
+Footage from at protest along the Flatbush Avenue Extension in Brooklyn shows several instances of police shoving protestors.
+
+The first clip shows a protestor speaking to an officer. The officer shoves him to the ground. Protestors crowd around as he stands. The officer then discharges pepper spray at him and at other protestors nearby.
+
+Additional clips show scuffles between police and protestors where police use their batons to repeatedly shove protestors.
+
+tags: pepper-spray, spray, shove, protestor
+
+id: ny-newyorkcity-88
+
+**Links**
+
+* https://int.nyt.com/data/videotape/finished/2020/07/1594404564/new_155-1254w.mp
```

Comment:
This link didn't work for me. I think this should be mp4

Reply:
Fixed!